### ISSUE TITLE
Find/search dialog for the "Show create table syntax" dialog or table info section

### ISSUE THREAD

**Issue by webtoolbox** [Sequel-Ace/Sequel-Ace#329]
Often times I find myself copy/pasting the entire schema of the table to search for text within it. It would be nice to be able to open a find/search box for the table SQL schema so that you can search the schema SQL directly in the app when you're on the Table Info tab or if you press Command+Option+S to see the schema.

**Comment by webtoolbox** [comment]
Why are feature requests being closed?

**Comment by Jason-Morcos** [comment]
> Why are feature requests being closed?

We have a stale bot that's designed to mark issues as "stale" if no activity has happened in a long time, and then close them eventually if nobody chimes in to the contrary. The bot wasn't working correctly, but we got it fixed again today so it started working as intended.
Really, this is just about the practicality of working on this project with such a small team. Feature requests are great, but if there are hundreds of feature requests and a handful of devs doing this part-time (after work) it's daunting. We really only have time to work on the features which are most meaningful to the community and, at the end of the day, honestly the ones we have the personal passion to pursue. However, we let feature requests, even ones we don't personally plan on implementing, stay open for some time to allow anyone in the community who _does_ have a particular passion about a feature to develop it and submit a pull request. We are incredibly happy to review and provide feedback on any PR from the community. However, if none of the core devs have the time/passion to pursue a feature, and nobody in the community jumps to develop the feature, we no longer benefit from leaving the issue open and it serves to only clutter the issue log and make it more difficult to identify more high-value features.

TL;DR: We fixed the stale bot which basically cleans up issues that have little to no activity and thus will probably not be implemented at this time.

**Comment by webtoolbox** [comment]
Got it. Thanks for the explanation.
